### PR TITLE
Multiselect n items selected

### PR DESCRIPTION
**Pull request by luizdanielmlima**
## Description

On the new select component (Select2), if more than 1 item is selected, it will show "N items selected" on the field.

### Desktop
image

## Types of changes

- [x] Fix (non-breaking change which fixes an issue)
- [ ] Feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Build (Changes that affect the build system or dependencies)
- [ ] Chore (Other changes that do not modify src or test files)
- [ ] CI (Changes to our CI configration and scripts)
- [ ] Docs (Changes to documentation files)
- [ ] Style (Changes to coding style, like linters and prettier)
- [ ] Refactor (Changes that do not add new functionality or alter current ones)
- [ ] Performance (Performance improvements)
- [ ] Test (Adding or changing tests or the testing setup)
- [ ] Revert (Reverting a previous change)

## Checklist:

- [ ] I have made sure my changes work properly on iOS devices running Safari.
- [ ] I have made sure my changes work properly on Android devices running Chrome.
- [x] I have made sure my changes work properly on Desktop.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

**Comment by logrock-eng** [comment]
# Netlify Preview Deployed 

 Preview: https://110--logrock-pebbles.netlify.app 

 ## View on Browserstack 

 https://live.browserstack.com/dashboard#os=iOS&os_version=15.0&device_browser=safari&zoom_to_fit=true&full_screen=true&url=https://110--logrock-pebbles.netlify.app&speed=1

File: lib/components/Select2/Select.tsx [hunk 1]
```diff
@@ -105,6 +135,10 @@ function Select<
         styles={customStyles}
         theme={selectTheme}
         inputId={selectID}
+        components={{
+          ...props.components,
+          ValueContainer: CustomValueContainer as any,
```
Comment (lipe-dev) [inline lib/components/Select2/Select.tsx:135]: if I'm not wrong, you can solve this by adding the Option type here
`CustomValueContainer<Option>`

and then on line 31:
const CustomValueContainer: FC<CustomValueContainerProps<OptionType>> = ({

Comment (lipe-dev) [reply lib/components/Select2/Select.tsx:135]: and also replace then any on line 28 with it

File: lib/components/Select2/Select.tsx [hunk 2]
```diff
@@ -19,6 +22,33 @@ import { CustomSelectProps } from "./Select.types";
 
 const defaultStatus = "info";
 
+const { ValueContainer } = components;
+
+interface CustomValueContainerProps extends ValueContainerProps {
+  getValue: () => Options<any>;
```
Comment (lipe-dev) [inline lib/components/Select2/Select.tsx]: isn't this function already available from the interface ValueContainerProps? this custom prop interface can probably be removed.

File: lib/components/Select2/Select.tsx (hunk 1 above)
Comment (luizdanielmlima) [reply lib/components/Select2/Select.tsx:135]: Thanks for the review, I will look into it

File: lib/components/Select2/Select.tsx (hunk 2 above)
Comment (luizdanielmlima) [reply lib/components/Select2/Select.tsx]: Yes, you're right, I removed the CustomValueContainerProps. But I couldn't make the changes you mentioned above work unfortunatelyf

**Comment by logrock-eng** [comment]
# Netlify Preview Deployed 

 Preview: https://110--logrock-pebbles.netlify.app 

 ## View on Browserstack 

 https://live.browserstack.com/dashboard#os=iOS&os_version=15.0&device_browser=safari&zoom_to_fit=true&full_screen=true&url=https://110--logrock-pebbles.netlify.app&speed=1

**Comment by logrock-eng** [comment]
# Netlify Preview Deployed 

 Preview: https://110--logrock-pebbles.netlify.app 

 ## View on Browserstack 

 https://live.browserstack.com/dashboard#os=iOS&os_version=15.0&device_browser=safari&zoom_to_fit=true&full_screen=true&url=https://110--logrock-pebbles.netlify.app&speed=1

**Comment by logrock-eng** [comment]
# Netlify Preview Deployed 

 Preview: https://110--logrock-pebbles.netlify.app 

 ## View on Browserstack 

 https://live.browserstack.com/dashboard#os=iOS&os_version=15.0&device_browser=safari&zoom_to_fit=true&full_screen=true&url=https://110--logrock-pebbles.netlify.app&speed=1

**Pull request merged**